curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.md

```diff
@@ -0,0 +1,122 @@
+---
+id: 5c6c06847491271903d37cfd
+title: Use the value attribute with Radio Buttons and Checkboxes
+challengeType: 0
+---
+
+## Description
+<section id='description'>
+When our form gets submitted, the form data sent to the server includes entries identifying the choices and input values. Inputs of type <code>radio button</code> and <code>checkbox</code> reports the value given to the value attribute.
+
+For example:
+<blockquote>
+&#60;label for="indoor"&#62; <br>&nbsp;&nbsp;&#60;input id="indoor" value="indoor" type="radio" name="indoor-outdoor"&#62;Indoor <br>&#60;/label&#62;<br>
+&#60;label for="outdoor"&#62; <br>&nbsp;&nbsp;&#60;input id="outdoor" value="outdoor" type="radio" name="indoor-outdoor"&#62;Outdoor <br>&#60;/label&#62;
+</blockquote>
+
+Here we have two grouped <code>radio button</code> inputs. When the user submits the form with the "indoor" <code>radio button</code> selected, the form data will include the line <code>indoor-outdoor=indoor</code>. This is the <code>name</code> attribute of <code>indoor-outdoor</code> and the <code>value</code> attribute of <code>indoor</code>.
```

Comment:
```suggestion
Here, we have two grouped <code>radio</code> inputs. When the user submits the form with the <code>indoor</code> option <code>radio</code> selected, the form data will include the line <code>indoor-outdoor=indoor</code>. This is the <code>name</code> attribute of <code>indoor-outdoor</code> and the <code>value</code> attribute of <code>indoor</code>.
```

Reply:
Reworded it a bit to make it look better and clearer, added option into `<code>`